index.html

```diff
@@ -0,0 +1,205 @@
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+        
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+               </div>
+            </div>
+          
+        </section>
+        <div class="background">
+             <div class="Primary"><h2 id="Primary-3">About Me</h2><img src="images/Icons/illustration1aboutme.png" class="frame" alt="about"></div>
+             <div class="supporting" id="support"> <p>I can help you build a product , feature or website Look through some of my work 
+                and experience! If you like what you see and have a project you need coded,<br> 
+                don’t hestiate to contact me.</p></div>
+                <div class="btn"><a href="#" class="Primary-btn"> Get My Resume</a></div>
+        </div>
+        <section id="section-3" class="grid">
+             <div  class="card-2">
+                 <div class="img-holder"><img src="images/Icons/icon-languages.png" alt="languages"></div>
+                    <h3>Languages</h3>
+                    <div class="cells">
+                        <a class="cell2" href="#"> Ruby on the rails</a>
+                        <a class="cell2" href="#"> Css</a>
+                        <a class="cell2" href="#"> Javascript</a>
+                        <a class="cell2" href="#"> Html</a>
+                   </div>
```

Comment:
Same thing here
- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 

```
 <ul class="cell2">
      <li> <a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a>></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
</ul>
```